crates/l2/utils/eth_client/mod.rs

```diff
@@ -814,7 +829,12 @@ impl EthClient {
                 get_gas_price = gas_price;
                 gas_price
             } else {
-                get_gas_price = self.get_gas_price().await?.as_u64();
+                let gas_price = self.get_gas_price().await?;
+                get_gas_price = if gas_price > u64::MAX.into() {
```

Comment:
Ditto.

Reply:
Done in [c7aa3b7](https://github.com/lambdaclass/ethrex/pull/1466/commits/c7aa3b7938549869411b412f53caf64eac0bbb2e)